yb-voyager/src/srcdb/data/gather-assessment-metadata/postgresql/table-columns-data-types.psql

```diff
@@ -5,7 +5,8 @@ SELECT
     c.table_name,
     c.column_name,
     CASE
-        WHEN c.data_type = 'USER-DEFINED' THEN c.udt_name
+        WHEN c.data_type = 'ARRAY' THEN c.udt_schema || '.' || SUBSTRING(c.udt_name FROM 2) || '[]'  -- Removes leading _ for array types and appends []
```

Comment:
what is use of adding `[]`

Reply:
To determine that its an array for Array of enums cases in the assessment report for live with ff/fb.